js/events/gesture/swipeable.js

```diff
@@ -20,13 +20,15 @@ const ACTION_TO_EVENT_MAP = {
     'onCancel': 'dxswipecancel'
 };
 
+const IMMEDIATE_TIMEOUT = 180;
```

Comment:
Why this timeout is equal 180?

Reply:
This value was declare in GestureEmitter